src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineTextSettingsTab.qml

```diff
@@ -67,13 +67,34 @@ FocusableItem {
 
         SeparatorLine { anchors.margins: -12 }
 
+        SpinBoxPropertyView {
+            id: beginLineGapFromTextControl
+            anchors.left: parent.left
+            anchors.right: parent.right
+            anchors.rightMargin: 2
+
+            titleText: qsTrc("inspector", "Gap between text and line")
+            propertyItem: root.model ? root.model.gapBetweenTextAndLine : null
+
+            step: 0.5
+            maxValue: 10.0
+            minValue: 0.0
+            decimals: 2
+
+            navigationName: "GapBetweenTextAndLine"
+            navigationPanel: root.navigationPanel
+            navigationRowStart: beginningTextOffsetSection.navigationRowEnd + 1
+        }
+
+        SeparatorLine { anchors.margins: -12 }
+
         TextSection {
             id: continuousTextSection
             titleText:  qsTrc("inspector", "Text when continuing to a new system")
             propertyItem: root.model ? root.model.continuousText : null
 
             navigationPanel: root.navigationPanel
-            navigationRowStart: beginningTextOffsetSection.navigationRowEnd + 1
+            navigationRowStart: beginningTextOffsetSection2.navigationRowEnd + 1
```

Comment:
I think you meant `beginLineGapFromTextControl` here :)
```suggestion
            navigationRowStart: beginLineGapFromTextControl.navigationRowEnd + 1
```

Reply:
Fixed, thanks for the catch!